index.html

```diff
@@ -1,24 +1,225 @@
 <!doctype html>
-<html class="no-js" lang="">
-    <head>
-        <meta charset="utf-8">
-        <meta http-equiv="x-ua-compatible" content="ie=edge">
-        <title>Muli</title>
-        <meta name="description" content="">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-
-        <link rel="manifest" href="site.webmanifest">
-        <link rel="apple-touch-icon" href="icon.png">
-        <!-- Place favicon.ico in the root directory -->
-
-        <link rel="stylesheet" href="assets/css/styles.css">
-    </head>
-    <body>
-        
-        
-
-        <script src="node_modules/jquery/dist/jquery.min.js"></script>
-        <script src="node_modules/zurb-foundation-5/js/foundation/foundation.js"></script>
-        <script src="assets/js/project.js"></script>
-    </body>
-</html>
\ No newline at end of file
+<html class="no-js" lang="english">
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="x-ua-compatible" content="ie=edge">
+  <title>Good Call</title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <link rel="manifest" href="site.webmanifest">
+  <link rel="apple-touch-icon" href="icon.png">
+  <!-- Place favicon.ico in the root directory -->
+
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:400,600,700" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.8.1/slick.min.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.8.1/slick-theme.min.css">
+  <link rel="stylesheet" href="assets/css/styles.css">
+</head>
+<body>
+
+  <div id="login-modal" class="reveal-modal modal  text-center"  data-reveal aria-labelledby="modalTitle" aria-hidden="true" role="dialog">
```

Comment:
More preference than anything, but usually this modal code would go right before the closing `</body>` tag